flang/include/flang/Optimizer/Dialect/FIROps.td

```diff
@@ -1789,6 +1789,24 @@ def fir_InsertValueOp : fir_OneResultOp<"insert_value", [NoSideEffect]> {
   }];
 }
 
+def fir_InsertOnRangeOp : fir_OneResultOp<"insert_on_range", [NoSideEffect]> {
+  let summary = "insert sub-value into a range on an existing sequence";
+
+  let description = [{
+    Insert a constant value into an entity with an array type. Returns a
```

Comment:
Oh, dang. This is an unrelated change that found its way into this.

Since you asked, this is intended to be an instruction used to initialize a range in a GlobalOp. So its values do have to be constants. (We may want to relax the inserted value to allow for symbolic addresses, which are load-time constant.)

Reply:
I will remove this from this PR since it is unrelated, etc.